packages/eslint-config-evg/config.js

```diff
@@ -0,0 +1,228 @@
+const ERROR = "error";
+const WARN = "warn";
+const OFF = "off";
+
+const errorIfStrict = process.env.STRICT ? ERROR : WARN;
+
```

Comment:
Yay!! can we also add:

`"prettier/prettier": errorIfStrict`

to make them show up as warnings in local development?